tests/scripts/create-dev-cluster.sh

```diff
@@ -138,9 +138,18 @@ show_usage() {
     echo "  -d value          Path to Rook examples directory (i.e github.com/rook/rook/deploy/examples)"
 }
 
+invocation_error() {
+ printf "%s\n" "$*" > /dev/stderr
+     	echo "$*" >& 2
```

Comment:
```suggestion
    printf "%s\n" "$*" > /dev/stderr
```

Reply:
@rkachach  wrote:
> @obnoxxx I think you forgot to remove the `echo` line. Plz, fix the indentation as well.

Indeed. I missed that. fixed.